src/Microsoft.TestPlatform.Common/Utilities/MetadataReaderHelper.cs

```diff
@@ -0,0 +1,190 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.Common.Utilities
+{
+    using System;
+    using System.Collections.Concurrent;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Reflection;
+    using System.Reflection.Metadata;
+    using System.Reflection.PortableExecutable;
+    using System.Text;
+
+    using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+
+    /* Expected attribute shape
+
+        namespace Microsoft.VisualStudio.TestPlatform
```

Comment:
I took a deeper look and the namespace is needed because we're searching for it , so I think it's good have a copy/paste sample.

```cs
private const string TestExtensionTypesAttributeV2 = "Microsoft.VisualStudio.TestPlatform.TestExtensionTypesV2Attribute";
```